### PR TITLE
Snippetizer tweaks

### PR DESCRIPTION
**Pull request by jglick**
@reviewbybees

### CODE REVIEW

File: cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly [hunk 1]
```diff
@@ -50,20 +50,21 @@ THE SOFTWARE.
...
+            <input type="button" value="Generate Groovy" onclick="handlePrototype_${id}(); return false" class="submit-button primary"/>
+            <f:textarea id="prototypeText_${id}" readonly="true"/>
         </f:block>
         <script>
-            function handlePrototype() {
+            function handlePrototype_${id}() {
                 buildFormTree(document.forms.config);
                 // TODO JSON.stringify fails in some circumstances: https://gist.github.com/jglick/70ec4b15c1f628fdf2e9 due to Array.prototype.toJSON
-                var json = Object.toJSON(JSON.parse(document.forms.config.elements.json.value).definition.snippetizer.prototype);
+                var json = Object.toJSON(JSON.parse(${configFormJS ?: 'document.forms.config'}.elements.json.value).${definitionJS ?: 'definition'}.snippetizer.prototype);
```
Comment (jglick) [inline cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly]: Generalizes the change in #40 to be more broadly usable.

**Comment by jglick** [comment]
All test failures are the notorious `ClosedByInterrupt` which @recampbell is trying to track down. Of course this masks any legitimate failures that might have come later.

**Comment by jglick** [comment]
@reviewbybees still awaiting review

**Comment by stephenc** [comment]
:+1:

**Pull request merged**